vertical-pod-autoscaler/recommender/Dockerfile

```diff
@@ -17,4 +17,4 @@ MAINTAINER Karol Kraskiewicz "karol.kraskiewicz@gmail.com"
 
 ADD recommender recommender
 
-CMD ./recommender --v=4 --stderrthreshold=info
+CMD ./recommender --v=4 --stderrthreshold=info --prometheus-address=http://my-release-prometheus-server.default.svc
```

Comment:
Can we pick some more reasonable prometheus server name and location? Like 
namespace: monitoring
service: prometheus
cc: @piosz

Reply:
SGTM though it's not a final commitment. Consider making this configurable via a flag.

Reply:
Done